packages/tenant-process/src/services/readModelService.ts

```diff
@@ -533,31 +557,14 @@ export function readModelServiceBuilder(
       });
     },
 
-    async getActiveDelegation({
-      eserviceId,
-      kind,
-    }: {
-      eserviceId: EServiceId;
-      kind: DelegationKind;
-    }): Promise<Delegation | undefined> {
-      const data = await delegations.findOne({
+    async getActiveProducerDelegationByEservice(
```

Comment:
Aligned the getDelegation as we are doing in the Incaricato branch, the get is specific for the kind we need to retrieve